src/main/java/uk/gov/justice/hmpps/prison/api/model/InternalMoveResult.java

```diff
@@ -13,7 +13,7 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-public class CellMoveResult {
+public class InternalMoveResult {
```

Comment:
This would be a breaking change for anyone that generates their client model from the Swagger docs. Have you checked if there is anyone doing this?

Reply:
I roll back this changes